src/aura/BGE_BatchGiftEntryTab/BGE_BatchGiftEntryTabHelper.js

```diff
@@ -15,6 +15,15 @@
             }
         });
         $A.enqueueAction(action);
+
+        const modalViewedKey = 'hasViewedRetModal';
```

Comment:
Not a necessary change, but could consider naming this `hasViewedRetirementPrompt` just to stay consistent with the other references to the prompt